### ISSUE TITLE
Offtank being grabbed by valkyres

### ISSUE THREAD

**Issue by Philliipe** [SunwellTracker/issues#2413]
Decription: Offtank being grabbed by valkyres

How it works: Offtank can be grabbed at the moment from val'kyrs.

How it should work: https://www.youtube.com/watch?v=h1x9MErf5NA&t=188s

3:25 he explains tanks cannot be grabbed by valkyres

Be a maintank or a offtank and you can clearly see you are able to be grabbed. 

Source (you should point out proofs of your report, please give us some source):

**Comment by Philliipe** [comment]
@Wojtyas What kind of help do you need? Is that for the other dev's or you need more information to be provided?

**Comment by Wojtyas** [comment]
@Philliipe I think it would be nice to have one more source here. Dev will take an eye on this anyway but since there are no two tanks necessary on LK during valkyr phase how LK should know who is the second tank. It can be done using script but still, it would be nice to have 2nd source just to make it clear.

**Comment by Philliipe** [comment]
@Wojtyas I'm kinda confused... You're saying that 2 tanks aren't necessary for LK 25heroic 2nd phase (val'kyr phase), how come? The main tank is taking LK for the soul reapers and the OFF TANK is taunting the val'kyrs from the air, so the discipline priest bubbles won't break up.

Anyways, I was grabbed many times as OFF TANK on LK 25hc and this shouldn't happen at all. 
Go ahead, test it whenever you can and come back to us. 

Looking forward,
Phillipe,
GM of Light Infusion

**Comment by Wojtyas** [comment]
Yeah, it is not necessary to switch tanks on every SR because of wall cds. And if cds are managed properly one tank can survive srs. 
How LK does know who is 2nd tank when first wave of valkyrs is spawned? There are no valkyrs floating over lk in this situation.

**Comment by Philliipe** [comment]
@Wojtyas How sindragosa knows that you're not a healer but you're protection paladin instead for the Unchained Magic (Instability stacks)? That's the developer's job, not mine.

I'm just comparing retail with Sunwell and reporting things that needs to be fixed. ;)

**Comment by Wojtyas** [comment]
Because there is a lot to heal on Sindragosa. But there might be nothing to offtank till valkyr are is spawned by the first time.

I'm glad you are reporting this stuff mate :)

**Comment by Philliipe** [comment]
@Wojtyas I hope you'll put effort in it and fix this.. Also the timers overlap valkyr's + defile.. This is insane and it shoudn't happen at all. 

Thank you for taking a look into this and i hope these things will be fixed very soon!
